src/concrete/ml/sklearn/tree_to_numpy.py

```diff
@@ -333,6 +345,121 @@ def tree_to_numpy(
     # but also rounding the threshold such that they are now integers
     q_y = tree_values_preprocessing(onnx_model, framework, output_n_bits)
 
-    _tree_inference, onnx_model = get_equivalent_numpy_forward_from_onnx(onnx_model)
+    _tree_inference, onnx_model = get_equivalent_numpy_forward_from_onnx(onnx_model, lsbs_to_remove)
 
     return (_tree_inference, [q_y.quantizer], onnx_model)
+
+
+# Remove this function once the truncate feature is released
+# FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4143
+def compute_lsb_to_remove_for_trees(onnx_model: onnx.ModelProto, q_x: numpy.ndarray) -> List[int]:
```

Comment:
if this function is temporary, I would make it private , so that we do not technically break the API once we remove it !